src/styles/partials/components/_input.scss

```diff
@@ -1,5 +1,3 @@
-$input-border-width: 2px;
-
 input,
 textarea {
 	border: $input-border-width solid $black;
```

Comment:
not sure how this built, without the variable declaration.

Reply:
this worked, because boostrap had the variable declared, changed to use our default border.